package.json

```diff
@@ -82,10 +86,14 @@
         }
       ]
     },
+    "moduleNameMapper": {
+      "(.+)\\.js": "$1"
+    },
```

Comment:
What is this?

Reply:
This is because flatbuffers is (or was) generating this kind of code:

`import { Foo } from 'foo.js'`

I'll check if it can be already removed.

Reply:
Removed via [78c6c1e](https://github.com/versatica/mediasoup/pull/927/commits/78c6c1ec920d61e272ae99c83e1f8e19f91bcc69)